app/BUILD.bazel

```diff
@@ -626,6 +626,7 @@ kt_android_library(
         artifact("androidx.viewpager2:viewpager2:1.0.0"),
         artifact("androidx.work:work-runtime-ktx:2.4.0"),
         artifact("com.caverock:androidsvg-aar"),
+        artifact("com.google.android:flexbox:2.0.1"),
```

Comment:
You'll need to also define this in the artifacts list in the WORKSPACE file otherwise Bazel won't know where to download it from. This is why the Bazel CI build is failing.

Reply:
Done